src/app/component/comments/components/delete-comment/delete-comment.component.scss

```diff
@@ -31,6 +31,16 @@
   }
 }
 
+@media screen and (max-width: 992px) {
+  .main-wrapper {
+    .cta-btn {
+      .btn-text {
+        font-size: 12px;
```

Comment:
we have these css styles in 3 places .. you can use dynamic font size - rem
or create a global class

Reply:
that situation is with other styles. I think, for that, we need to refactor all comment block (move buttons from own components to commentList component). It is more than that local bug task.